src/assets/styles/styles/custom/_dropdown.scss

```diff
@@ -37,3 +37,8 @@
   position: relative;
   top: -43px;
 }
+
+.display-main-button-outlined {
+  position: relative;
+  top: -45px;
```

Comment:
jeżeli przycisk jest typu outlined, to jest o 2px wyższy (po 1px na ramkę z każdej strony), więc bez tej poprawki będzie się rozjeżdżać